io-engine/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -277,7 +277,10 @@ impl<'n> Nexus<'n> {
         let paused = self.as_mut().pause_rebuild_jobs(uri).await;
 
         let idx = match self.children_iter().position(|c| c.uri() == uri) {
-            None => return Ok(()),
+            None => {
```

Comment:
would it be easier to move 277 below getting idx? Then we can just return Ok() here.

Reply:
Probably yes, but I wouldn't touch that now.